bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerCreateOp.java

```diff
@@ -92,6 +93,42 @@
     }
 
     /**
+     * Constructor
```

Comment:
LedgerCreateOp is package private. I don't think you need to overload the constructor.